src/test/java/integration/PostcodePoolIT.java

```diff
@@ -96,15 +99,15 @@ public void createPostcodePool() throws Exception {
                         body("postcode", equalTo(postCode)).
                 extract().
                         response();
-        long timeFinished = currentTimeMillis();
+        LocalDateTime timeFinished = now();
 
         // check created data
         PostcodePool createdPostcodePool = postcodePoolService.getEntityByUuid(UUID.fromString(response.path("uuid")));
-        long timeCreated = createdPostcodePool.getCreated().getTime();
-        long timeModified = createdPostcodePool.getLastModified().getTime();
+        LocalDateTime timeCreated = createdPostcodePool.getCreated();
+        LocalDateTime timeModified = createdPostcodePool.getLastModified();
 
-        assertTrue(WRONG_CREATED_MESSAGE, timeFinished >= timeCreated && timeCreated >= timeStarted);
-        assertTrue(WRONG_LAST_MODIFIED_MESSAGE, timeFinished >= timeModified && timeModified >= timeStarted);
+        assertTrue(WRONG_CREATED_MESSAGE, timeFinished.isAfter(timeCreated) && timeCreated.isAfter(timeStarted));
+        assertTrue(WRONG_LAST_MODIFIED_MESSAGE, timeFinished.isAfter(timeModified) && timeModified.isAfter(timeStarted));
```

Comment:
Plz, change. isAfter is not enough. It should do ">=".
Plz, create helper method assertDateBetween(date, dateFrom, dateTo) in the new class AssertHelper(package util).